### PR TITLE
Handlers reference documentation

### PR DESCRIPTION
**Pull request by palourde**
Closes https://github.com/sensu/sensu-docs/issues/244

### CODE REVIEW

File: content/sensu-core/2.0/reference/handlers.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+-------------|------
+description  | The handler command to be executed. The event data is passed to the process via `STDIN`._NOTE: the `command` attribute is only supported for Pipe handlers (i.e. handlers configured with `"type": "pipe"`)._
+required     | true (if `type` equals `pipe`)
+type         | String
+example      | {{< highlight shell >}}"command": "/etc/sensu/plugins/pagerduty.rb"{{< /highlight >}}
+
+env_vars      | 
+-------------|------
+description  | An array of environment variables to use with command execution._NOTE: the `env_vars` attribute is only supported for Pipe handlers (i.e. handlers configured with `"type": "pipe"`)._
+required     | true (if `type` equals `pipe`)
```
Comment (mercul3s) [inline content/sensu-core/2.0/reference/handlers.md]: I think env_vars default to required false.

File: content/sensu-core/2.0/reference/handlers.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+  event handlers, making it easy to manage groups of actions that should be
+  executed for certain types of events.
+
+### Pipe handlers
+
+Pipe handlers are external commands that can consume [event][3] data via STDIN.
+
+#### Pipe handler command
+
+Pipe handler definitions include a `command` attribute which is literally an
```
Comment (mercul3s) [inline content/sensu-core/2.0/reference/handlers.md]: I'd suggest rewording this slightly:
Pipe handler definitions include a `command` attribute, which is a command to be executed by the Sensu backend.

**Review by mercul3s — APPROVED** [review]
Have a couple minor notes, but overall this looks great!

File: content/sensu-core/2.0/reference/handlers.md (hunk 1 above)
Comment (palourde) [reply content/sensu-core/2.0/reference/handlers.md]: You are completely right, good catch!
